packages/base/rules-snapshot.json

```diff
@@ -0,0 +1,503 @@
+{
+  "@typescript-eslint/brace-style": "off",
```

Comment:
are all rules listed in each package? Is that for completeness to ensure that if we accidently, for example, add the typescript package into the base that we'll see it flipped on? Would it be better to only list the ones specific for this package and see that in snapshots something is added? I probably just don't understand the method of how this is generated yet.

Reply:
or is it that prettier turns all these rules off! (ding ding ding) nvm me